include/SMaterialLayer.h

```diff
@@ -7,7 +7,7 @@
 

```

Comment:
why havent we nuked this entire file yet?

Reply:
@Crisspl ?

Reply:
because ill do it later, why do you so much want to nuke files now at the moment? XD

Reply:
less code = make code review easier

Reply:
nuked resolved